.pre-commit-config.yaml

```diff
@@ -0,0 +1,69 @@
+default_language_version:
+    python: python3
+
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.31.1
+    hooks:
+    -   id: pyupgrade
+        args: [--py38-plus]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: setup.cfg
+      - id: end-of-file-fixer
+      - id: check-yaml
+        args: ['--allow-multiple-documents']
+      - id: debug-statements
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.9.0
+    hooks:
+      - id: rst-inline-touching-normal
+  - repo: https://github.com/ambv/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        exclude: ^docs/
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        additional_dependencies: [ 'flake8-rst-docstrings' ]
+        args: ['--config=setup.cfg']
+  - repo: https://github.com/timothycrosley/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ['--profile', 'black', '--filter-files']
+        exclude: ^docs/
+  - repo: https://github.com/pycqa/pydocstyle
+    rev: 6.1.1
+    hooks:
+      - id: pydocstyle
+        args: ['--convention=numpy', '--match="(?!test_).*\.py"']
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.26.3
+    hooks:
+      - id: yamllint
+        args: ['--config-file', '.yamllint.yaml']
+  # - repo: https://github.com/mgedmin/check-manifest
+  #   rev: "0.48"
+  #   hooks:
+  #     - id: check-manifest
```

Comment:
```suggestion
  - repo: https://github.com/mgedmin/check-manifest
    rev: "0.48"
    hooks:
      - id: check-manifest
```

Reply:
This was commented by @aulemahal because it caused issues previously. Is it fine now?

Reply:
Indeed, `check-manifest` 0.48 is incompatible with git < 2.11. Doris and nérée have git 1.8.